src/app.js

```diff
@@ -25,6 +25,10 @@ var OffsetToLineColumnConverter = require('./lib/offsetToLineColumnConverter')
 
 var examples = require('./app/example-contracts')
 
+// components
+var fileExplorer = require('_file-explorer')
```

Comment:
For the moment I'd hope to keep it just in `src/lib/`. Once interfaces are stable we could think about moving these to separate modules (please check #404 for a direction we were taking).